src/spikeinterface/sorters/internal/spyking_circus2.py

```diff
@@ -3,7 +3,7 @@
 import os
 import shutil
 import numpy as np
-import os
+import psutil
```

Comment:
is this used anywhere? It needs to be added to the requirements full in any case

Reply:
Nope, this has been removed